_rules/SC1-2-1-video-only-description-track.md

```diff
@@ -0,0 +1,107 @@
+---
+name: Video only element description track
+description: |
+  This rule checks that description tracks that come with non-streaming `video` elements, without audio, are descriptive.
```

Comment:
The expectation is that the descriptive track is in the same language of the video. This is not mentioned in the rule's description. Shouldn't it be?

Reply:
Wilco and I think this would be to much detail for the description.